info.json

```diff
@@ -19,20 +19,20 @@
     "help": "https://github.com/fortinet-fortisoar/solution-pack-fortimanager-ztp-framework/blob/release/1.0.2/README.md",
```

Comment:
update version from "1.0.2" to "1.0.3" in help link

Reply:
done